src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_northrend_beasts.cpp

```diff
@@ -500,10 +502,49 @@ struct npc_snobold_vassal : public ScriptedAI
         }
     }
 
+    Player* SelectNewVictim()
+    {
+        const Map::PlayerList& players = me->GetMap()->GetPlayers();
+        std::vector<Player*> availablePlayers;
+        for (auto i = players.begin(); i != players.end(); ++i)
+        {
+            if (!i->GetSource()->IsAlive())
+            {
```

Comment:
according to our coding guidelines we do not use any curly braces { } for oneliners